cloud_functions/climateiq_data_export_cf/main.py

```diff
@@ -279,6 +319,8 @@ def _calculate_h3_indexes(
     study_area_metadata: dict,
     chunk_metadata: dict,
     spatialized_predictions: pd.DataFrame,
+    bucket_name: str,
+    object_name: str,
 ) -> pd.Series:
     """Projects cell centroids to H3 indexes.
```

Comment:
What's the difference between a cell and a row?

Reply:
the row contains the entire data- prediction, cell centroid, h3 info, etc